app/models/version.rb

```diff
@@ -0,0 +1,13 @@
+class Version < ApplicationRecord
+  belongs_to :assessment
+  belongs_to :format
+  belongs_to :created_by, class_name: 'UserProfile'
+
+  has_many :assets, inverse_of: :version
+  has_many :version_tags
+  has_many :tags, through: :version_tags
+
+  enum visiblity: %i[internal external]
```

Comment:
What is `visibility`?

Reply:
whether the AssessmentVerions (or whatever we end up calling it) can be viewed by the public or only by members.  Originally I had this called "public/private" but those conflict with ruby keywords 😁